docs/2.guide/3.going-further/6.nuxt-app.md

```diff
@@ -19,6 +19,10 @@ In Nuxt 2, this was referred to as **Nuxt context**.
 Jump over the `NuxtApp` interface documentation.
 ::
 
+## The Nuxt context
+
+Many composables and utilities, both built-in and user-made, may require access to the Nuxt instance. You may get an error stating that "A composable that requires access to the Nuxt instance was called outside of a plugin, Nuxt hook, Nuxt middleware, or Vue setup function." when trying to use a certain composable (for example, [useState](/docs/api/composables/use-state)). That is because that specific composable needs to access the Nuxt context, which doesn't exist everywhere on your application. Currently, the Nuxt context is only accessible in [plugins](/docs/guide/directory-structure/plugins), [Nuxt hooks](/docs/guide/going-further/hooks), [Nuxt middleware](/docs/guide/directory-structure/middleware), and [setup functions](https://vuejs.org/api/composition-api-setup.html) (in pages and components).
```

Comment:
```suggestion
Many composables and utilities, both built-in and user-made, may require access to the Nuxt instance.

You may get an error stating that 'A composable that requires access to the Nuxt instance was called outside of a plugin, Nuxt hook, Nuxt middleware, or Vue setup function.' when trying to use a composable (for example, [useState](/docs/api/composables/use-state)). That is because that specific composable needs to access the Nuxt context, which doesn't exist everywhere on your application.

Currently, the Nuxt context is only accessible in [plugins](/docs/guide/directory-structure/plugins), [Nuxt hooks](/docs/guide/going-further/hooks), [Nuxt middleware](/docs/guide/directory-structure/middleware), and [setup functions](https://vuejs.org/api/composition-api-setup.html) (in pages and components). You can also explicitly call functions within this context by using [`nuxtApp.runWithContext`](/docs/api/composables/use-nuxt-app#runwithcontext).
```